rllib/algorithms/sac/torch/sac_torch_learner.py

```diff
@@ -323,6 +319,12 @@ def _update_module_target_networks(
         """Updates the target Q network(s) of a module.
 
         Applies Polyak averaging for the update.
+
```

Comment:
Same here, move this description to the super (abstract) method and remove the docstring here entirely.